client/src/views/asset-list.js

```diff
@@ -1,12 +1,37 @@
 import Snabbdom from 'snabbdom-pragma'
-
+import { formatAssetAmount } from './util'
 import layout from './layout'
 
-export default ({ assetMap, t, ...S }) => {
+const staticRoot = process.env.STATIC_ROOT || ''
+export default ({ assetMap, assetList, loading, t, ...S }) => {
+
+  const { assets, query, total } = !assetList ? {} : assetList
+  const { start_index, sort_dir, sort_field, limit } = !query ? {} : query
+  const pageLink = `/assets/registry?sort_field=${encodeURIComponent(sort_field)}&sort_dir=${encodeURIComponent(sort_dir)}`
```

Comment:
Link URLs should not begin with a leading `/` so that they'll be relative to the `<base href>` and not to the root domain.

This makes a difference when esplora is served from a subdirectory and not the `/` root path, like on the live website. If  `https://blockstream.info/liquid/` links to `/assets/registry`, it'll be relative to the root and lead to `https://blockstream.info/asset/registry/` without `liquid`.

Reply:
changes have been made